src/main/webapp/edit.html

```diff
@@ -1,13 +1,33 @@
-<!-- Page displays 'my organizations' or 'organizations' based on if user are maintainer or moderator -->
-<!DOCTYPE html>
+<!-- <!DOCTYPE html>
```

Comment:
Is there a reason to leave this file if everything is commented out? Will we need to use it in the future?

Reply:
will delete since not being used